docs/atomic-migrate.1.md

```diff
@@ -0,0 +1,57 @@
+% ATOMIC(1) Atomic Man Pages
+% Shishir Mahajan
+% October 2015
+# NAME
+atomic-migrate - Migrate container's content from one backend storage to another.
+
+# SYNOPSIS
+**atomic migrate COMMAND [OPTIONS]**
+
+atomic migrate allows the user to easily migrate images, volumes, and 
+containers from one version of atomic to another. With this command, 
+users can quickly save all their data from the current atomic instance, 
+change the container's content storage backend, and then import all their old data 
+to the new system.
+
+# COMMANDS
+**export**
+
+export command will export all the current images, volumes, and containers
+to the specified directory, in the /images, /volumes, /containers subdirectories.
```

Comment:
Are you worried about space.  Please add it here also.  specified directory, /var/lib/atomic/migrate by default.  Content will be placed in the /images, /volumes, /containers subdirectories.